### PR TITLE
Added carpool command

### PR DESCRIPTION
**Pull request by glowcone**
(See #4)
The function seems a bit verbose though - is there a lodash trick to minimize it?

### CODE REVIEW

File: bin/hubot [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 set -e
 
-npm install
+#npm install
```
Comment (colindean) [inline bin/hubot]: I'm pretty sure this should not be commented out...

**Comment by justinxreese** [comment]
Good call removing that extra file.

I think I agree with @colindean on that `npm install` change, although I'm not sure how crucial that is. At worst it's redundant, so I don't want to remove it without looking into it more. Can you remove that change so we can merge the rest, @recurza?

**Comment by glowcone** [comment]
Woops! I temporarily commented it since I was restarting hubot many times, and forgot to uncomment it before commit

**Comment by justinxreese** [comment]
Awesome thanks!

As far as making it smaller, I think it's reasonably sized. Not sure lodash would help significantly 👍 good work.

**Comment by justinxreese** [comment]
We use this in our public team at http://codeandsupply.co/chat if you'd like to join our community. It's mostly local, but the conversations are definitely built to cross borders

**Pull request merged**